DevCycle.SDK.Server.Local/Api/LocalBucketing.cs

```diff
@@ -373,19 +426,57 @@ private Function GetFunction(string name)
 
         private int GetParameter(string param)
         {
-            const int objectIdString = 1;
-
-            // ReSharper disable once InconsistentNaming
-            var __new = GetFunction("__new");
-
             var paramAddress =
-                (int)__new.Invoke(Encoding.Unicode.GetByteCount(param), objectIdString)!;
+                (int)NewFunc.Invoke(Encoding.Unicode.GetByteCount(param), WasmObjectIdString)!;
 
             Encoding.Unicode.GetBytes(param, WASMMemory.GetSpan(paramAddress, Encoding.Unicode.GetByteCount(param)));
 
             return paramAddress;
         }
+        
+        /// <summary>
+        /// Writes the bytes to WASM memory as a Uint8Array object and returns the header pointer
+        /// </summary>
+        /// <param name="paramData">An array of bytes to set in WASM memory</param>
+        /// <returns>A WASM memory pointer</returns>
+        private int GetUint8ArrayParameter(byte[] paramData)
```

Comment:
This function needs a good review. I ported what the Go SDK does and it works but I I'm not sure I'm doing the pin/unpin correctly